src/components/ProfileDropdown.jsx

```diff
@@ -16,16 +10,17 @@ import NewIcon from '@material-ui/icons/NewReleases';
 import BookmarkIcon from '@material-ui/icons/Bookmark';
 import DoneIcon from '@material-ui/icons/Done';
 import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+import { EXPERIMENTAL_VERSION } from '../config/constants';
 
 // hacky way to override Menu style
-const StyledMenu = withStyles((theme) => ({
+const StyledMenu = withStyles({
   paper: {
     border: '1px solid black',
     width: '350px',
-    //background: theme.palette.background.default,
+    background: '#F3F6FA',
```

Comment:
Hardcoded values like this are hard to change later on if the theme changes. Change this to be `theme.background.default` and instead update `theme.js` so that the background color is this value.

Reply:
Ok I changed the default background value to the `#F3F6FA` & also instead of what you recommended above it did not work so I changed it to be `theme.palette.background.default`